src/types/mod.rs

```diff
@@ -33,6 +33,30 @@ pub enum IonType {
     Struct,
 }
 
+impl fmt::Display for IonType {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                IonType::Null => "null",
+                IonType::Boolean => "boolean",
+                IonType::Integer => "integer",
+                IonType::Float => "float",
+                IonType::Decimal => "decimal",
+                IonType::Timestamp => "timestamp",
+                IonType::Symbol => "symbol",
+                IonType::String => "string",
+                IonType::Clob => "clob",
+                IonType::Blob => "blob",
+                IonType::List => "list",
+                IonType::SExpression => "sexp",
+                IonType::Struct => "struct",
+            }
+        )
+    }
+}
```

Comment:
Out of curiosity, where do you need this?  Is the `Debug` not sufficient for some use case?

Reply:
I keep doing println-debugging and `{}` is just natural. I can remove it if you want..

Reply:
No biggie--I was just wondering.